admin/bin/init-dev-storage.js

```diff
@@ -19,68 +19,69 @@ try {
 } catch (error) {
   console.error(error)
 }
-const azure = require('azure-storage')
-const bluebird = require('bluebird')
+const { TableClient } = require('@azure/data-tables')
+const { QueueServiceClient } = require('@azure/storage-queue')
 const names = require('../../deploy/storage/tables-queues.json')
 
 if (!process.env.AZURE_STORAGE_CONNECTION_STRING) {
   process.exitCode = -1
   console.error('env var $AZURE_STORAGE_CONNECTION_STRING is required')
 }
 
+const connectionString = process.env.AZURE_STORAGE_CONNECTION_STRING
+
 const queueNames = names.queues
 const tableNames = names.tables
 const poisonQueues = queueNames.map(q => q + '-poison')
 const allQueues = queueNames.concat(poisonQueues)
-const tableService = getPromisifiedService(azure.createTableService())
-const queueService = getPromisifiedService(azure.createQueueService())
+
+async function deleteTableEntries (tableName) {
```

Comment:
Should we actually be calling our own interface here, rather than the MS implementation directly?

Reply:
done